### ISSUE TITLE
How long will the session be kept?

### ISSUE THREAD

**Issue by McwilliamsJacobson** [fuck-xuexiqiangguo/Fuck-XueXiQiangGuo#197]
Would like to add auto-run script to run the software so don’t need to log again?

**Comment by stonebreaker2019** [comment]
I don't think it can be achieved. It's based on the web version. You still need to scan the QR-code to log in. Like some other functions (auto forwards and add to the favor) can't be achieved.
